### DISCUSSION TITLE
Unable to install SO-network properly

### DISCUSSION THREAD

**Discussion by Leiruz** [Security-Onion-Solutions/securityonion#4336] [Unsupported Versions] [answered]
Hello,

We are a group of final-year students at Ngee Ann Polytechnic. We are looking to deploy Security Onion as a proof of concept and we have faced many issues with installing Security Onion. We read the documentation and tried installing it from the ISO file but it keeps crashing halfway with issues related to Salt Minion & Salt Master not working. We decided to run it on CentOS7 instead and we managed to successfully install SO-Install, but we were stuck at running the SO-network installation. The installation is stuck at 95% currently.

Our server met the installation requirement for Evaluation mode and attached below is the log file from Security Onion and network diagram of what we're trying to accomplish as proof of concept.

Do feel free to email me at _S10193642@connect.np.edu.sg_ if you need further details. Thank you so much to the Security Onion team for such a wonderful product. Thank you, everyone, in advance for your help

**Attached Documents**
![image](https://user-images.githubusercontent.com/50233597/120059182-681dd880-c082-11eb-87e9-fae9335b127c.png)
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/6564054/sosetup.log)

**Comment by defensivedepth** [comment, marked as answer]
Hello there!

First thing I would try would be to disable the wifi interface and then run a fresh install again, this time with the ISO. If it errors out, upload the sosetup.log again and we will take a look.

Having two interfaces with active IPs could be the cause of the issue, though setup should have warned about that.

**Reply by Leiruz** [reply]
Hi Josh,

Thank you so much for responding. We are unable to install it multiple times on the server via LAN connection as the cable is connected to a Next-Gen Firewall and we suspect that some files such as the virus signature file belonging to the IDS might have been blocked when it is being downloaded from Security Onion. Hence we had to resort to using a hotspot to download Security Onion.

**Reply by defensivedepth** [reply]
Ok, so if you download the ISO through the hotspot, then disconnect the hotspot/wifi and try to install using the ISO?

Again - the key issue is to make sure there are not more than 1 NICs that have IPs on them.
